delta-lake/delta-spark341db/src/main/scala/com/databricks/sql/transaction/tahoe/rapids/GpuDeltaDataSource.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022-2025, NVIDIA CORPORATION.
```

Comment:
This is a new file, right?
```suggestion
 * Copyright (c) 2025, NVIDIA CORPORATION.
```

Reply:
Same comment as above